### PR TITLE
[Units] Add MK⁻¹ unit alias

### PR DESCRIPTION
**Pull request by andrewfg**
See discussion https://github.com/openhab/openhab-addons/issues/17638#issuecomment-2444023328

Signed-off-by: AndrewFG <software@whitebear.ch>

### CODE REVIEW

**Comment by andrewfg** [comment]
@maniac103 for info..

**Comment by ccutrer** [comment]
Should we also be adding `mirek` as an alias? And perhaps changing the proper name of the unit to `MK⁻¹ `, with `mired` as the alias?

**Comment by andrewfg** [comment]
^
I would support that.

File: bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/Units.java [hunk 1]
```diff
@@ -319,6 +319,10 @@ public final class Units extends CustomUnits {
         SimpleUnitFormat.getInstance().alias(MONTH, "mo");
         SimpleUnitFormat.getInstance().alias(WEEK, "wk");
         SimpleUnitFormat.getInstance().alias(YEAR, "y");
+
+        // other aliases
+        SimpleUnitFormat.getInstance().alias(MIRED, "mired");
+        SimpleUnitFormat.getInstance().alias(MIRED, "mirek");
```
Comment (jlaur) [inline bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/Units.java:325]: These could probably be moved up right after line 296? All the others have their alias right after the label, except for the few works-arounds just above.

Comment (andrewfg) [reply bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/Units.java]: Ok. Done.

**Comment by lolodomo** [comment]
I followed the discussion and it looks good to me too.

**Comment by lolodomo** [comment]
If I have a String `myUnit` containing a unit, what is the code to use to check that is a unit in "mired" whatever the accepted symbol ?
If I use `Units.MIRED.toString().equals(myUnit)`, this will probably work only with "mired" with the current code and only "MK⁻¹" when this PR is merged.

**Comment by andrewfg** [comment]
> If I have a String myUnit containing a unit, what is the code to use to check that is a unit in "mired" whatever the accepted symbol ?

Umm. From memory I believe it is the following. .. But perhaps I should add a Junit test to confirm that works?

```
Unit unit = QuantityType.valueOf(yourString).getUnit();
```

**Comment by andrewfg** [comment]
@lolodomo I just added the Junit test (it passes).

**Comment by ccutrer** [comment]
Or `UnitUtils.parseUnit` if you only have the unit, without a scalar for a full quantity

**Comment by lolodomo** [comment]
> Or `UnitUtils.parseUnit` if you only have the unit, without a scalar for a full quantity

Oh  yes, looks better, thank you.

**Pull request merged**